### PR TITLE
Add "The practice for more benefit"

### PR DESCRIPTION
**Pull request by AmeniSassi**
Please review this, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2433)

### CODE REVIEW

File: arabic/practice.md [hunk 1]
```diff
@@ -0,0 +1 @@
+تطبيق ما تعلمته من هذا الدرس في أعمالي سيكون الخطوة المقبلة لترسيخ المعلومات و الإستفادة
```
Comment (casey-collab) [inline arabic/practice.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look.

File: arabic/practice.md (hunk 1 above)
Comment (casey-collab) [inline arabic/practice.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline arabic/practice.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: arabic/practice.md (hunk 1 above)
Comment (cbuckey-uda) [inline arabic/practice.md]: @AmeniSassi, it looks like your fix was good, but now the file you moved exists in both places. You'll need to delete the old one using `git rm`.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**